teamA/lib/Views/course_list.dart

```diff
@@ -0,0 +1,252 @@
+import 'package:flutter/material.dart';
+import 'package:namer_app/main.dart';
+
+final ColorScheme customColorScheme = ColorScheme(
+  primary: Color(0xFF6A5A99), // Purple color as primary
+  onPrimary: Color(0xFFFFFFFF), // White text on primary
+  primaryContainer: Color(0xFFEDE6FF), // Light lavender
+  onPrimaryContainer: Color(0xFF2D004A), // Dark purple
+  secondary: Color(0xFF6A5A99), // Using similar purple as secondary
+  onSecondary: Colors.white, // Black text on background
+  surface: Color(0xFFFFFFFF), // White surface color
+  onSurface: Colors.black, // Black text on surface
+  error: Colors.red, // Error color red
+  onError: Colors.white, // White on error
+  brightness: Brightness.light, // Light mode
+);
+
+// This is the course list UI
+
+class MyApp extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      debugShowCheckedModeBanner: false,
+      home: CourseList(),
+      theme: ThemeData(
+        colorScheme: customColorScheme,
+        useMaterial3: true,
+      ),
+      title: 'Learning Lens',
+    );
+  }
+}
+
+class CourseList extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+        debugShowCheckedModeBanner: false,
+        home: Scaffold(
+            appBar: AppBar(
+              backgroundColor: Theme.of(context)
+                  .colorScheme
+                  .primaryContainer, // Use primary container color
+              elevation: 0,
+              flexibleSpace: SafeArea(
+                  child: Row(
+                mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                children: [
+                  IconButton(
+                    icon: Icon(Icons.arrow_back),
+                    onPressed: () {
+                      Navigator.push(
+                        context,
+                        MaterialPageRoute(builder: (context) => DevLaunch()),
+                      );
+                      //to do something once pressed
+                    },
+                  ),
+                  Expanded(
+                    child: Align(
+                      alignment: Alignment.center,
+                      child: Text(
+                        'Learning Lens',
+                        style: TextStyle(
+                          color:
+                              Theme.of(context).colorScheme.onPrimaryContainer,
+                          fontSize: 20,
+                          fontWeight: FontWeight.bold,
+                        ),
+                      ),
+                    ),
+                  ),
+                ],
+              )),
+            ),
+            body: Padding(
```

Comment:
Add a SingleChildScrollView here. While it's unlikely that anyone will have enough courses for vertical space to be an issue, better safe than sorry.